### PR TITLE
Restore previous timezone tests

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Since the change is soon to be reverted

### CODE REVIEW

File: tests/test_on_data/test_format_converters/test_lightningpose_converter.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def assertNWBFileStructure(self, nwbfile_path: str, stub_test: bool = False):
         with NWBHDF5IO(path=nwbfile_path) as io:
             nwbfile = io.read()
 
-            self.assertEqual(nwbfile.session_start_time, datetime(2023, 11, 9, 10, 14, 37))
+            self.assertEqual(nwbfile.session_start_time, datetime(2023, 11, 9, 10, 14, 37)).astimezone())
```
Comment (CodyCBakerPhD) [inline tests/test_on_data/test_format_converters/test_lightningpose_converter.py:140]:
```suggestion
            self.assertEqual(nwbfile.session_start_time, datetime(2023, 11, 9, 10, 14, 37).astimezone())
```

File: tests/test_on_data/test_format_converters/test_miniscope_converter.py [hunk 2]
```diff
@@ -170,7 +170,7 @@ def assertNWBFileStructure(self, nwbfile_path: str):
 
             self.assertEqual(
                 nwbfile.session_start_time,
-                datetime(2021, 10, 7, 15, 3, 28, 635),
+                datetime(2021, 10, 7, 15, 3, 28, 635).astimezone()),
```
Comment (CodyCBakerPhD) [inline tests/test_on_data/test_format_converters/test_miniscope_converter.py:173]:
```suggestion
                datetime(2021, 10, 7, 15, 3, 28, 635).astimezone(),
```

File: tests/test_on_data/test_format_converters/test_spikeglx_converter.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def assertNWBFileStructure(self, nwbfile_path: FilePath):
         with NWBHDF5IO(path=nwbfile_path) as io:
             nwbfile = io.read()
 
-            expected_session_start_time = datetime(2020, 11, 3, 10, 35, 10)
+            expected_session_start_time = datetime(2020, 11, 3, 10, 35, 10).astimezone())
```
Comment (CodyCBakerPhD) [inline tests/test_on_data/test_format_converters/test_spikeglx_converter.py:36]:
```suggestion
            expected_session_start_time = datetime(2020, 11, 3, 10, 35, 10).astimezone()
```

**Review by h-mayorquin — APPROVED** [review]
What happened? saw the dev etests failing.

**Comment by CodyCBakerPhD** [comment]
It was dev then main

PyNWB reverted the `datetime` without `timezone` thing because it caused problems with DANDI publication

**Pull request merged**